src/elements/table/Table.js

```diff
@@ -86,10 +88,11 @@ export class NovoTableHeader {
                                 </div>
                             </div>
                             <!-- FILTER DROP-DOWN -->
-                            <novo-dropdown side="right" *ngIf="column.filtering" class="column-filters">
+                            <novo-dropdown side="right" *ngIf="column.filtering" class="column-filters" [showAfterSelect]="column.range || column.multiple">
```

Comment:
No I mean the "input" for that property is only on the <item> component, not the <novo-dropdown>. You have it below as well on the <item> for the custom.